src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { Dispatch, useEffect } from 'react';
+import cn from 'classnames';
+import { Errors } from '../types/ErrorType';
+
+type Props = {
+  errorMessage: string;
+  setErrorMessage: Dispatch<React.SetStateAction<Errors>>;
```

Comment:
u can do it easier(up to u)
```suggestion
  setErrorMessage: (error: Errors) => void;
```